pkg/scheduler/api/job_info.go

```diff
@@ -160,22 +161,24 @@ func NewTaskInfo(pod *v1.Pod) *TaskInfo {
 	preemptable := GetPodPreemptable(pod)
 	revocableZone := GetPodRevocableZone(pod)
 	topologyInfo := GetPodTopologyInfo(pod)
+	easAcceleratorFlavorId := GetPodEasAcceleratorInfo(pod)
 
 	jobID := getJobID(pod)
 
 	ti := &TaskInfo{
-		UID:           TaskID(pod.UID),
-		Job:           jobID,
-		Name:          pod.Name,
-		Namespace:     pod.Namespace,
-		Priority:      1,
-		Pod:           pod,
-		Resreq:        resReq,
-		InitResreq:    initResReq,
-		Preemptable:   preemptable,
-		BestEffort:    bestEffort,
-		RevocableZone: revocableZone,
-		NumaInfo:      topologyInfo,
+		UID:                    TaskID(pod.UID),
+		Job:                    jobID,
+		Name:                   pod.Name,
+		Namespace:              pod.Namespace,
+		Priority:               1,
+		Pod:                    pod,
+		Resreq:                 resReq,
+		InitResreq:             initResReq,
+		Preemptable:            preemptable,
+		BestEffort:             bestEffort,
+		RevocableZone:          revocableZone,
+		NumaInfo:               topologyInfo,
+		EasAcceleratorFlavorId: easAcceleratorFlavorId,
```

Comment:
task also need `EasAccelerator bool`, if EasAcceleratorFlavorId != "", task's EasAccelerator  is true